incubator/foundry-vtt/templates/ingress-nginx.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (eq .Values.ingress.enabled true) (eq .Values.ingress.class "nginx") }}
+{{- if and (eq .Values.ingress.enabled true) (ne .Values.ingress.class "contour") }}
```

Comment:
Why use not contour over == nginx? Is there are any special case that eq nginx don't cover?

Just sanity checking

Reply:
Yes, we use Azure AKS with ingress addon the the class name is: `addon-http-application-routing`

So it would be good to have this configurable. 

https://docs.microsoft.com/en-us/azure/aks/http-application-routing